src/typings/react-native.d.ts

```diff
@@ -240,6 +240,9 @@ declare module 'react-native' {
         accessible?                      : boolean;
         onAcccessibilityTap?             : Function;
 
+        // desktop
+        valueNow?                        : number;
```

Comment:
Is it somehow used in Native?
I see a property being added for View for Web, but no action for Native.
If it's Web-only let's have it ariaValueNow for the consistency with other attributes. And I'll be easier to clean up later, when the API improvements will come.

Reply:
Updated to ariaValueNow.